module-2/backend/week_two.md

```diff
@@ -4,29 +4,43 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+1. At a high level, what is ActiveRecord? What does it do/allow you to do? 
+  It is a Ruby Library that allows you to work with relational SQL/
 2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+  They are associations (would they be considered class methods? An example would be that a car belongs to one driver but a     driver could have many cars.
```

Comment:
This is kind of a wonky question: I think the point we were trying to make is that the methods that ActiveRecord gives you when you create these relationships (e.g. being able to call `car.driver`) must be called on an *instance* of Car. You wouldn't be able to call `Car.driver`, because `.driver` is looking for the driver of a particular car.